.github/workflows/generate-docs-for-dbt-core.yml

```diff
@@ -19,149 +19,160 @@ defaults:
-            .changes/unreleaded/*.yml
+            .changes/unreleased/*.yaml
           if-no-files-found: error
           retention-days: 15
 
-  # create branch on dbt-core
-  create_dbt_core_branch:
+  commit_dbt_core:
     needs: [set_variables, build_index]
     runs-on: ubuntu-latest
 
     steps:
-      - name: Check out repository code
+      - name: Check out dbt-core main branch
         uses: actions/checkout@v3
         with:
           repository: dbt-labs/dbt-core
           ref: main
-          # token: PAT?
+          token: ${{ secrets.FISHTOWN_BOT_PAT }}
  
       - name: Create branch
         run: |
-          git checkout -b ${{needs.set_variables.outputs.branch_name}}
-          git push -u origin ${{needs.set_variables.outputs.branch_name}}
+          git checkout -b ${{ needs.set_variables.outputs.branch_name }}
+          git push -u origin ${{ needs.set_variables.outputs.branch_name }}
       
-      - name: Check current branch
+      - name: Print branch status
         run: |
           git status
 
-  commit_dbt_core:
-    needs: [set_variables, create_dbt_core_branch]
-    runs-on: ubuntu-latest
-
-    steps:
-
-      - name: Check out the dbt-core branch
-        uses: actions/checkout@v3
+      - name: Download index artifact
+        uses: actions/download-artifact@v3
         with:
-          repository: dbt-labs/dbt-core
-          ref: ${{ needs.set_variables.outputs.branch_name }}
-          # token: PAT?
-
-      - uses: actions/download-artifact@v3
-        with:
-          name: ${{needs.set_variables.outputs.index_artifact_name}}
+          name: ${{ needs.set_variables.outputs.index_artifact_name }}
           path: "core/dbt/include"
       
-      - uses: actions/download-artifact@v3
+      - name: Download changelog artifact(s)
+        uses: actions/download-artifact@v3
         with:
-          name: ${{needs.set_variables.outputs.changie_artifact_name}}
+          name: ${{ needs.set_variables.outputs.changie_artifact_name }}
           path: ".changes/unreleased"
 
       - name: Print branch status
         run: |
           git status
 
+      # note that if nothing has changed with the repo, the index file won't be
+      # committed since there are no changes, even if there are changelogs.
       - name: Commit & Push changes
         run: |
           git config user.name 'Github Build Bot'
           git config user.email 'buildbot@fishtownanalytics.com'
           git pull
           git add .
-          git commit -m "Add new index.html and changelog yaml files for dbt-docs"
+          git commit -m "Add new index.html and changelog yaml files from dbt-docs"
           git push
 
       - name: Create Pull Request
-        run:
-          gh pr create --title ${{ needs.set_variables.outputs.pr_title }} \
-                       --body ${{ needs.set_variables.outputs.pr_body }} \
+        run: |
+          gh pr create --title "${{ needs.set_variables.outputs.pr_title }}" \
```

Comment:
Feels like these `${{ needs.set_variables.outputs.pr_title }}` are also jinja ish templates?

Reply:
Is this just an observation?  I'm not sure if you're asking something I need to answer 😄

Reply:
Just an observation!